protostar/testing/fuzzing/strategies/short_strings.py

```diff
@@ -0,0 +1,26 @@
+import string
+
+from hypothesis.strategies import SearchStrategy, text
+from starkware.cairo.lang.compiler.ast.cairo_types import CairoType, TypeFelt
+
+from protostar.testing.fuzzing.exceptions import SearchStrategyBuildError
+from protostar.testing.fuzzing.strategy_descriptor import StrategyDescriptor
+
+
+def string_to_bytes_int(val: str) -> int:
+    if not val:
+        return 0
+    # pylint: disable=consider-using-f-string
+    return int("0x" + "".join("{:02x}".format(ord(c)) for c in val), 16)
+
+
+class ShortStringsStrategyDescriptor(StrategyDescriptor):
+    def build_strategy(self, cairo_type: CairoType) -> SearchStrategy[int]:
+        if not isinstance(cairo_type, TypeFelt):
+            raise SearchStrategyBuildError(
+                "Strategy 'short strings' can only be applied to felt parameters."
+            )
+
+        return text(alphabet=list(string.ascii_lowercase), min_size=0, max_size=31).map(
```

Comment:
I would use all of the ascii here not only lowercase

Reply:
Yes, sure, how about https://docs.python.org/3/library/string.html#string.printable?

Reply:
Done in [1a1dc66](https://github.com/software-mansion/protostar/pull/886/commits/1a1dc66b90aed39d3e2737a1131b7e819748495e)